src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -446,6 +457,11 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
                             Title = title
                         }
                     };
+                    var popupWindow = newStep.stepUIPopup as PopupWindow;
+                    if(popupWindow != null)
+                    {
+                        popupWindow.WebBrowserUserDataFolder = userDataFolder != null ? userDataFolder.FullName : string.Empty;
+                    }
```

Comment:
Why do we need this code here? Shouldn't this be set one time only at the Welcome step?

Reply:
@RobertGlobant20 
1st question:
- we need to set the WebBrowserUserDataFolder  before calling the WebView2.EnsureCoreWebView2Async() method so it can set the cache folder in the right location, I put it here because in GuidesManager we have access to DynamoViewModel.Model.PathManager.

2nd question.
- I did a change that will be set only for popups that use html, check the commit below.

3rd question.
- Not all the Welcome steps use html then it will be set only for the ones using html (and by consequence WebView2).

commit:
https://github.com/DynamoDS/Dynamo/pull/13245/commits/b37dcc167897b8190e41c0a1985117abdae61855

Reply:
@RobertGlobant20 are there any MSDN docs on how these webview2 startup options affect other instances of WebView2?

Reply:
@mjkkirschner MS documentation doesn't talk specifically about that but I think the startup options are specific for each WebView2 instance.
If you question is because guided tours and documentationbrowser will be using the same cache - user data folder (UDF) folder, MS Documentation says "It's possible for your host app to overlap them" but doesn't say anything about the consequences, I just opened the "User Interface" tour that opens the DocumentationBrowser at the same time than a Popup opened but I didn't see any problem (the Popup is not using html for this case), then once Notification Center is completed we will need to see what happen if we open the Notifications Center and the DocumentationBrowser at the same time.
https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=win32

Reply:
@RobertGlobant20 thanks- I'm also curious because inside of Hosts like Revit, they will use WebView2 and so will the other addins running in the Host process. It's the wild west!

Reply:
@mjkkirschner well, we asked Revit about it but for my understanding they are still in the process of adopting WebView2 then they haven't deal with this feature yet so if Revit team decide to use another cache folder in AppData for WebView2 we can evaluate a change in Dynamo to use the same than them, another thing is that we need to ask if this feature of enable/disable cache for WebView2 will be enabled in the future (since I didn't find anything about it).
Also I checked with AutoCAD team and they said they create the folder in the user profile (consider that Autocad is using C++).
Thanks